src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString
+    .split(';')
+    .filter(property => property.trim())
+    .reduce((acc, property) => {
+      const [key, value] = property.split(':');
+
+      if (key && value) {
+        acc[key.trim()] = value.trim();
+      }
+
+      return acc;
+    }, {});
+
+  return styleArray;
```

Comment:
If you want to make changes by calling them in a chain, you don't need to create a separate variable and you can immediately return the result.